.changeset/poor-pans-chew.md

```diff
@@ -0,0 +1,14 @@
+---
+"@aws-amplify/ui-react": patch
+---
+
+ui-react(fix): Update ShowPasswordButton to use role=“switch” and add additional SR context
+
+User Story
+- As a screen reader user, I want the Show Password button to announce its current context, so that I know when it is toggled or not.
+
+Acceptance Criteria
```

Comment:
nit: since it'll be a part of our release note, we may want to remove "Acceptance Criteria"